llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -985,8 +1013,9 @@ processInputModule(std::unique_ptr<Module> M) {
 
   // Instrument each image scope device globals if the module has been
   // instrumented by sanitizer pass.
-  if (isModuleUsingAsan(*M))
-    Modified |= runModulePass<SanitizeDeviceGlobalPass>(*M);
+  if (isModuleUsingAsan(*M)) {
+    Modified |= runSanitizerPasses(*M);
```

Comment:
Suggest to add extra lit tests to cover sycl-post-link tool changes.

Reply:
Changes in sycl-post-link was reverted.